src/app/views/comments/_comment.html.erb

```diff
@@ -18,7 +18,7 @@
           <% end if can?(comment, :destroy) %>
 
           <br>
-          <%= simple_format comment.content %>
+          <%= raw(to_html(comment.content)) %>
```

Comment:
Instead of using `raw` use the same technique used in `render_code` to sanitize the output and make it safe for rendering into the page